scripts/scraper-ng/rules/html-require-recipe-ingredients/ingredient-handlers/data-examples.js

```diff
@@ -0,0 +1,90 @@
+const select = require("hast-util-select");
+const visit = require("unist-util-visit");
+
+const utils = require("./utils.js");
+
+/**
+ * Function returning true only if the given node is a text node
+ * that contains only newlines.
+ */
+function isNewlineOnlyTextNode(node) {
+  return node.type === "text" && node.value.match(/^\n*$/);
+}
+
+/**
+ * Checks a single example. This only does any checking for live samples:
+ * that is, examples that contain a call to the EmbedLiveSample macro.
+ */
+function checkExample(body, exampleNodes, logger) {
```

Comment:
I think `body` is unused here

```suggestion
function checkExample(exampleNodes, logger) {
```